### ISSUE TITLE
JDK16 : java/lang/invoke/MethodHandles/classData/ClassDataTest.java error: <T>classData(Lookup,String,Class<T>) is not public in MethodHandles; cannot be accessed from outside package

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#11366]
Failure link
------------

From an internal build `Test_openjdknext_j9_sanity.openjdk_x86-64_linux_Nightly/363`:
```
21:51:12  openjdk version "16-internal" 2021-03-16
21:51:12  OpenJDK Runtime Environment (build 16-internal+0-adhoc.jenkins.BuildJDKnexts390xlinuxNightly)
21:51:12  Eclipse OpenJ9 VM (build master-9f11871a18a, JRE 16 Linux s390x-64-Bit Compressed References 20201202_402 (JIT enabled, AOT enabled)
21:51:12  OpenJ9   - 9f11871a18a
21:51:12  OMR      - a58cd2f5687
21:51:12  JCL      - f7021059fd3 based on jdk-16+26)
```
To rebuild the failed tests in <Jenkins URL>=https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder, use the following links:
22:41:17  https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/parambuild/?JDK_VERSION=16&JDK_IMPL=openj9&BUILD_LIST=openjdk&PLATFORM=s390x_linux&TARGET=jdk_lang_j9_0

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------
```
21:52:36  --------------------------------------------------
21:52:42  TEST: java/lang/invoke/MethodHandles/classData/ClassDataTest.java

21:52:42  /home/jenkins/workspace/Test_openjdknext_j9_sanity.openjdk_x86-64_linux_Nightly/openjdk-tests/openjdk/openjdk-jdk/test/jdk/java/lang/invoke/MethodHandles/classData/ClassDataTest.java:64: error: <T>classData(Lookup,String,Class<T>) is not public in MethodHandles; cannot be accessed from outside package
21:52:42          int value = MethodHandles.classData(lookup, "_", int.class);
21:52:42                                   ^
21:52:42    where T is a type-variable:
21:52:42      T extends Object declared in method <T>classData(Lookup,String,Class<T>)

21:52:42  Note: Recompile with -Xlint:unchecked for details.
21:52:42  19 errors
21:52:42  
21:52:42  TEST RESULT: Failed. Compilation failed: Compilation failed
```

fyi @tajila @babsingh

**Comment by babsingh** [comment]
Related: https://github.com/eclipse/openj9/issues/11321#issuecomment-738080469. For the OpenJ9 MH impl, this failure can be fixed by changing `MethodHandles.classData` to a public method in JDK16. The OpenJ9 MH impl change is being avoided since it will be obsolete. The plan is to resolve this failure automatically once OpenJDK JSR292 support (#7352) is enabled for JDK16.

**Comment by JasonFengJ9** [comment]
Note that these tests have been excluded via https://github.com/AdoptOpenJDK/openjdk-tests/pull/2168

**Comment by babsingh** [comment]
This can be closed after #11815 and #11816 are merged. I have opened https://github.com/eclipse/openj9/issues/11817 for the new failures seen with OpenJ9. The new failures do not need to be fixed for the JDK16 0.25 release.
